flytekit/clis/sdk_in_container/run.py

```diff
@@ -644,7 +644,8 @@ def list_commands(self, ctx):
         return [str(p) for p in pathlib.Path(".").glob("*.py") if str(p) != "__init__.py"]
 
     def get_command(self, ctx, filename):
-        ctx.obj[RUN_LEVEL_PARAMS_KEY] = ctx.params
+        if ctx.obj:
```

Comment:
Added this condition cause it fails otherwise when building API docs with the error: `NoneType object does not support item assignment`.